.vscode/extensions.json

```diff
@@ -3,7 +3,9 @@
     "charliermarsh.ruff",
     "ms-python.mypy-type-checker",
     "ms-python.python",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
+    "redhat.vscode-yaml",
```

Comment:
This is the official YAML extension.
It's useful if we want to have features other than validation like YAML autocomplete. But if not, it seems redundant to me given that we are relying on `yamllint`.